Zilon.Core/Zilon.Core.Tests/Persons/HumanPersonTests.cs

```diff
@@ -1,1018 +1,1025 @@
-            hpStat.Value.Should().Be(EXPECTED_PERSON_HP);
-        }
-
-        /// <summary>
-        /// Тест проверяет, что если экипировать предмет бонусом к здоровью,
-        /// то здоровье персонажа остаётся текущим.
-        /// </summary>
-        [Test]
-        public void HumanPerson_EquipPropWithHungerResistance_DownPassOfSatietyDescreased()
-        {
-            // ARRANGE
-
-            const int START_PERSON_HP = 10;
-            const int EXPECTED_DOWNPASS = 2;
-
-            var personScheme = new TestPersonScheme
-            {
-                Hp = START_PERSON_HP,
-                Slots = new[]{
-                    new PersonSlotSubScheme
-                    {
-                        Types = EquipmentSlotTypes.Aux
-                    }
-                },
-
-                SurvivalStats = new[]
-                {
-                    new TestPersonSurvivalStatSubScheme
-                    {
-                        Type = PersonSurvivalStatType.Satiety,
-                        MinValue = -100,
-                        MaxValue = 100,
-                        StartValue = 0
-                    },
-
-                    new TestPersonSurvivalStatSubScheme
-                    {
-                        Type = PersonSurvivalStatType.Hydration,
-                        MinValue = -100,
-                        MaxValue = 100,
-                        StartValue = 0
-                    }
-                }
-            };
-
-            var defaultAct = new TestTacticalActScheme
-            {
-                Stats = new TestTacticalActStatsSubScheme
-                {
-                    Efficient = new Roll(6, 1)
-                }
-            };
-
-            var evolutionData = CreateEvolutionFakeModule();
-
-            var survivalRandomSourceMock = new Mock<ISurvivalRandomSource>();
-            var survivalRandomSource = survivalRandomSourceMock.Object;
-
-            var person = new HumanPerson(personScheme, defaultAct, evolutionData, survivalRandomSource);
-
-            var armorPropScheme = new TestPropScheme
-            {
-                Equip = new TestPropEquipSubScheme
-                {
-                    Rules = new[]
-                    {
-                        new PersonRule(EquipCommonRuleType.HungerResistance, PersonRuleLevel.Normal)
-                    },
-                    SlotTypes = new[]
-                    {
-                        EquipmentSlotTypes.Aux
-                    }
-                }
-            };
-
-            var satietyStat = person.GetModule<ISurvivalModule>().Stats.SingleOrDefault(x => x.Type == SurvivalStatType.Satiety);
-
-            var armorProp = new Equipment(armorPropScheme);
-
-            // ACT
-            person.GetModule<IEquipmentModule>()[0] = armorProp;
-
-            // ASSERT
-            satietyStat.DownPassRoll.Should().Be(EXPECTED_DOWNPASS);
-        }
-    }
-}
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5480188)